recipes/vowpalwabbit/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "vowpalwabbit" %}
+{% set version = "8.7.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://scgrahamarchive.blob.core.windows.net/vowpal-wabbit/vowpal_wabbit_2ee2bcf_rapidjson.tgz 
+  sha256: befed5ecd42decd533b8e11a98f0e76f9d48951c085b52786277d86ec06923ac
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake>=3.13
```

Comment:
There needs to be a single space after the package name, e.g. `cmake >=3.13`

Reply:
got it